studentInfo.py

```diff
@@ -0,0 +1,72 @@
+  fullItems = ['Abyssal Mask','Adaptive Helm','Archangels Staff','Ardent Censer','Unholy Grail','Banner of Command','Banshees Veil','Blade of the Ruined King','Dead Mans Plate','Deaths Dance','Duskblade of Drakathaar','Edge of Night','Essence Reaver','Eye of Equinox','Eye of the Oasis','Eye of the Watcher','Face of the Mountain','Frost Queens Claim','Frozen Heart','Frozen Mallet','Gargoyle Stoneplate','Guardian Angel','Guinsoos Rageblade','Hextech GLP-800','Hextech Gunblade','Hextech Protobelt-01','Iceborn Gauntlet','Infinity Edge','Knights Vow','Liandrys Torment','Lich Bane','Locket of the Iron Solari','Lord Dominiks Regards','Ludens Echo','Manamune','Maw of Malmortius','Mejais Soulstealer','Mercurial Scimitar','Mikaels Crucible','Morellonomicon','Mortal Reminder','Nashors Tooth','Ohmwrecker','Phantom Dancer','Rabadons Deathcap','Randuins Omen','Rapid Firecannon','Ravenous Hydra','Redemption','Righteous Glory','Rod Of Ages','Ruby Sightstone','Runaans Hurricane','Rylais Crystal Scepter','Spirit Visage','Statikk Shiv','Steraks Gage','Sunfire Cape','Talisman of Ascension','The Black Cleaver','The Bloodthirstier','Thornmail','Titanic Hydra','Trinity Force','Void Staff','Warmogs Armor','Wits End','Yomuus Ghostblade','Zekes Convergence','Zhonyas Hourglass','ZZ-Rot Portal','','','','','','','','']
+  def __init__(self):
+    self.assignRandomK()
+    self.assignRandomD()
+    self.assignRandomA()
+    self.randomChampion()
+    self.randomLane()
+    self.randomItem1()
+    self.randomItem2()
+    self.randomItem3()
+    self.randomItem4()
+    self.randomItem5()
+    self.randomItem6()
+
+  def randomLane(self):
+     self.lane = random.choice(self.lanes)
+
+  def randomChampion(self):
+     self.mainChamp = random.choice(self.mainChamps)
+  
+  def assignRandomK(self):
+     self.K = random.randint(0,30)
+  def assignRandomD(self):
+     self.D = random.randint(0,30)
+  def assignRandomA(self):
+     self.A = random.randint(0,30)  
+  def randomItem1(self):
+    self.item1 = random.choice(self.fullItems)
+  def randomItem2(self):
+    self.item2 = random.choice(self.fullItems)  
+  def randomItem3(self):
+    self.item3 = random.choice(self.fullItems)
+  def randomItem4(self):
+    self.item4 = random.choice(self.fullItems)
+  def randomItem5(self):
+    self.item5 = random.choice(self.fullItems)
+  def randomItem6(self):
+    self.item6 = random.choice(self.fullItems)
+
+inputQuestions = [ 
+  "For CHAMPIONS, type 0",
+]
+
+def getUserSelection():
+  print (inputQuestions[0])
+  return int(input("Type selection and press enter: "))
+
+def RandomizedChampions():
+  count = int( input("How Many Randomized Champions? "))
+  for i in range(0, count):
+    champ = Champion()
+    print (str(champ.mainChamp) + " " + str(champ.lane) + ", " + str(champ.item1) + ", " + str(champ.item2) + ", " + str(champ.item3) + ", " + str(champ.item4) +", " + str(champ.item5) + ", " + str(champ.item6))
```

Comment:
I'd recommend having `RandomizedChampions()` return a list of randomized champions, because on line 3 through 5 you declare `students = [Champion()]`, which is never actually used.

If `RandomizedChampions()` were to return a list of randomized champions then you could store the result of that in `students` (after renaming students to something like `champions`).

Then I'd recommend having a function whose sole input is a list of champions, that loops through the supplied list of champions and prints the details about them, and calling it after populating the `champions` list.

This would make it easier to add features to the program, as the state of the program would be centralized in `main()`.